storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -540,6 +549,19 @@ func (c *PrometheusConverter) addTimeSeriesIfNeeded(lbls []prompb.Label, startTi
 	}
 }
 
+// handleStartTime adds a zero sample 1 millisecond before ts iff startTs == ts.
```

Comment:
```suggestion
// handleStartTime adds a zero sample 1 millisecond before ts if startTs == ts.
```

Reply:
@ArthurSens "iff" is intentional here, it's conventinonal notation for "if and only if".

Reply:
Oh TIL